src/components/Dayentry.js

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+
+const StyledDatum = styled.div`
+  border: solid 1px;
+  width: 100px;
+  background-color: #f28549;
+  border-color: #f28549;
+  border-radius: 5px;
+  padding: 9px;
+  margin-bottom: 5px;
+  color: #fae9c3;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans',
+    'Helvetica Neue', sans-serif;
+  font-size: 15px;
+`;
+
+const StyledBox = styled.div`
+  border: solid 1px;
+  border-color: #e86d1b;
+  background-color: #e86d1b;
+  border-radius: 5px;
+  padding: 15px;
+  margin-bottom: 25px;
+  color: #fae9c3;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans',
+    'Helvetica Neue', sans-serif;
+  font-size: 15px;
+`;
+
+const Dayentry = props => {
+  return (
+    <>
+      <StyledDatum>{props.datum}</StyledDatum>
+      <StyledBox>{props.text}</StyledBox>
+    </>
```

Comment:
You can destructure your object as follows:
```suggestion
const Dayentry = {datum, text} => {
  return (
    <>
      <StyledDatum>{datum}</StyledDatum>
      <StyledBox>{text}</StyledBox>
    </>
```

Reply:
sometimes, like here or with extra div tags, it helps me for "my own structure/my style" and to keep tracking for later on for me as a beginner. like how "bad" is it? not a good style, but will I get problems later on maybe? @stefankorder